app/jobs/update_plage_ouverture_expiration_job.rb

```diff
@@ -0,0 +1,8 @@
+class UpdatePlageOuvertureExpirationJob < ApplicationJob
+  def perform
+    today = Date.today
+    PlageOuverture.where(expired: false).each do |po|
```

Comment:
I don't think this works because the column name is `expired_cached`. which makes me say: please add a spec for this job :)